documentation/src/main/wiki/Assembler.md

```diff
@@ -0,0 +1,137 @@
+#Fragment Assembler
+Fragment Assembler joins all Fragments into the final output. It occurs after Knots processing and
+just before the response to the site visitor.
+
+##How does it work?
+Fragment Assembler reads Knot Context having Fragments, joins all Fragments into one output, updates 
+Knot Context and returns back to the caller. See examples below for more details.
+
+###How are Fragments joined?
+Lets explain this join process with the example. Fragment assembler reads Knot Context having 
+three Fragments:
+```html
+<html>
+<head>
+  <title>Test</title>
+</head>
+<body>
+<h1>test</h1>
+```
+```html
+  <h2>this is webservice no. 1</h2>
+  <div>message - a</div>
+```
+```html
+</body>
+</html>
+```
+Fragment Assembler joins all those Fragments into one output:
+```html
+<html>
+<head>
+  <title>Test</title>
+</head>
+<body>
+<h1>test</h1>
+  <h2>this is webservice no. 1</h2>
+  <div>message - a</div>
+</body>
+</html>
+```
+###How does Assembler join unprocessed Fragments?
+Lets imagine that some Fragments were not processed and they still contain dynamic snippets definitions. 
+It is not expected behaviour so Fragment Assembler must handle it. There are three possible strategies 
+provided: `SIMPLE`, `EXTRACT`, `CLEAR`. They can be configured with entry `assemblyStrategy`.
```

Comment:
I'd change names of strategies as this:
- SIMPLE into AS_IS
- EXTRACT into UNWRAP
- CLEAR into IGNORE